build.sbt

```diff
@@ -54,7 +55,22 @@ lazy val commonSettings = Seq(
 )
 
 lazy val root = Project("geotrellis", file(".")).
-  dependsOn(raster, vector, proj4, spark, sparkEtl, s3, accumulo).
+  dependsOn(
```

Comment:
Recently realized that `aggregate` is more appropriate here: http://stackoverflow.com/questions/26477388/is-the-use-of-aggregate-following-by-dependson-redundant-with-the-same-modul

That would also mean that including % "test" shouldn't be required.